admin-dev/themes/new-theme/public/theme.css

```diff
@@ -12747,3 +12747,8 @@ span.twitter-typeahead {
   width: 100% !important; }
   span.twitter-typeahead input.tt-input {
     width: 100% !important; }
+
+.stock-app table td {
```

Comment:
Hello @Azouz-Jribi 
It's not here to change the css to the vue application. It's directly in the vue file. 
@nihco2, can you explain better please!
Regards